bitnami/kubeapps/templates/shared/config.yaml

```diff
@@ -2,10 +2,10 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ template "kubeapps.kubeops-config.fullname" . }}
+  name: {{ template "kubeapps.clusters-config.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
-    app.kubernetes.io/component: kubeops
+    app.kubernetes.io/component: kubeapps
```

Comment:
If it's a generic ConfigMap shared by every Kubeapps component, shouldn't we simply remove the `app.kubernetes.io/component` label?

Reply:
Yes, I thought it was mandatory/recommended somehow and I just used "kubeapps" as a generic name. Let's remove it. Thanks!